sklearn/tests/test_impute.py

```diff
@@ -486,44 +486,25 @@ def test_imputation_copy():
     # made, even if copy=False.
 
 
-def test_chained_imputer_rank_one():
```

Comment:
This should still be valid if we run multiple imputations and take the mean, shouldn't it? Let's try leave the tests in.

Reply:
It still is: I changed the name to `test_iterative_imputer_rank_one` and moved it to be with the other `functional` tests.